src/main/java/dev/dkvl/womutils/panel/WomPanel.java

```diff
@@ -141,7 +132,9 @@ public void mouseClicked(MouseEvent e)
         add(createOverViewPanel(), c);
         c.gridy++;
 
-        add(lastUpdated.getLabel(), c);
+        MiscInfoLabel lastUpdated = new MiscInfoLabel(MiscInfo.LAST_UPDATED);
```

Comment:
Why is this done separately and not in `createOverviewPanel`?

Reply:
The last updated label is added under the overview panel with a different formatting. Would it be better to completely separate it from the overview?